code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -112,6 +112,7 @@ Des: Removes all infected images from the alien.
 			var/searchfor = "infected"
 			if(findtext(I.icon_state, searchfor, 1, length(searchfor) + 1))
 				qdel(I)
+				client.images -= I
```

Comment:
I think you should remove first before `qdel(I)`

Reply:
Aight, fixed it. This is what I get for making PRs while sleep deprived.